lib/tapjoy/ldap/api/user.rb

```diff
@@ -0,0 +1,112 @@
+require 'memoist'
+module Tapjoy
+  module LDAP
+    module API
+      module User
+        class << self
+          extend Memoist
+          def create(fname, lname, type, group)
+            # Properly capitalize names
+            fname, lname = [fname, lname].map(&:titleize)
```

Comment:
I think we could use the #capitalize method here and save having to require `active_support/core_ext/string/inflections` in lib/tapjoy/ldap.rb